climpred/toy.py

```diff
@@ -0,0 +1,259 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+from climpred.bootstrap import bootstrap_compute
+from climpred.prediction import compute_perfect_model
+from climpred.tutorial import load_dataset
+
+# standard setup
+lead = xr.DataArray(np.arange(0, 20, 1), dims='lead')
+lead['lead'] = lead.values
+
+control = load_dataset('MPI-control-1D')['tos'].isel(period=-1, area=1)
+# amplitudes of the signal and noise
+noise_amplitude = control.std().values * 2.5
+signal_amplitude = control.std().values
+# period of potentially predictable variable
+signal_P = 8
```

Comment:
I find it odd to have these defaults defined up here. Maybe it would be cleaner to do something like:

```python
DEFAULTS = {
    'lead': lead = xr.DataArray(np.arange(0, 20, 1), dims='lead'),
}
```

and so on down the line. Then for each function in here, set things `=None` that could use a default and if they go through as `None` just assign the default from the dictionary.